### PR TITLE
Translate QM headers to ECEF coordinate system.

### PR DESCRIPTION
**Pull request by fester**

### CODE REVIEW

File: src/QuantizedMeshIO.cpp [hunk 1]
```diff
@@ -295,6 +298,25 @@ static void write_qmheader(BinaryIO& bio,
     bio.write_double(qmheader.horizon_occlusion.z, e);
 }
 
+bool point_to_ecef(Vertex& p)
+{
+    OGRSpatialReference mercator;
+    OGRSpatialReference ecef;
+
+    mercator.importFromEPSG(3857);
+    ecef.importFromEPSG(4978);
```
Comment (adrpar) [inline src/QuantizedMeshIO.cpp]: is it really necessary to allocate and initiate those SpatialReferences on every conversion of a point? Cant those be put on the class and just initialized once?

File: src/QuantizedMeshIO.cpp [hunk 2]
```diff
@@ -295,6 +298,25 @@ static void write_qmheader(BinaryIO& bio,
...
+
+    std::unique_ptr<OGRCoordinateTransformation> tr(
+        OGRCreateCoordinateTransformation(&mercator, &ecef));
+
+    if(tr == nullptr)
+    {
+        return false;
+    }
```
Comment (adrpar) [inline src/QuantizedMeshIO.cpp]: I think the same applies for the transformation

File: src/QuantizedMeshIO.cpp [hunk 3]
```diff
@@ -313,6 +335,12 @@ bool write_mesh_as_qm(const std::shared_ptr<FileLike>& f,
     // Write QM Header
     Vertex c = (bbox.max + bbox.min) / 2.0;
 
+    if(!point_to_ecef(c))
+    {
```
Comment (adrpar) [inline src/QuantizedMeshIO.cpp]: ah... I see... this is only done once. Then forget about the comments above :)

File: src/QuantizedMeshIO.cpp (hunk 1 above)
Comment (fester) [reply src/QuantizedMeshIO.cpp]: This function is called once per tile, hence I don't think adding an extra class is worth it.

**Pull request merged**